core/templates/dev/head/pages/email_dashboard/EmailDashboardResult.js

```diff
@@ -40,6 +38,16 @@ oppia.controller('EmailDashboardResult', [
         $scope.invalid.body = true;
         invalidData = true;
       }
+      return invalidData;
+    };
+
+    $scope.submitEmail = function() {
+      var resultHandlerUrl = UrlInterpolationService.interpolateUrl(
+        RESULT_HANDLER_URL, {
+          query_id: getQueryId()
+        });
+      var invalidData = validateEmailSubjectAndBody();
```

Comment:
This is just returning a boolean, so call it dataIsInvalid. Otherwise the name suggests that you have an object of some sort here.